import unittest
from alarmexception import AlarmException

class TestAlarmException(unittest.TestCase):
     def test_alarmexceptio(self):
        #  self.assertEqual(, )
        
if __name__ == '__main__':
    unittest.main()